app/assets/javascripts/discourse/app/helpers/slice.js

```diff
@@ -0,0 +1,13 @@
+import { registerRawHelper } from "discourse-common/lib/helpers";
+
+registerRawHelper("slice", slice);
```

Comment:
Let's drop this line - we don't need this helper to be accessible in `.hbr` files